package.json

```diff
@@ -9,7 +9,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "angular-mocks": "^1.6.1",
+    "angular-mocks": "^1.7.0",
```

Comment:
Any specific reason to choose 1.7.0 when we have 1.7.8?

Reply:
@galipremsagar yes, we cannot use `angular 1.7.0` with `angular-mocks 1.7.8`, the version of both of them should be matched.